cogs/deadline.py

```diff
@@ -81,20 +82,16 @@ async def timenow_error(self, ctx, error):
     #          indicating that the reminder has been added
     # -----------------------------------------------------------------------------------------------------------------
     @commands.command(name="addhw",
```

Comment:
Should be instructor only

```suggestion
    @commands.has_role('Instructor')
    @commands.command(name="addhw",
```